### ISSUE TITLE
bug: re-login causes error 500

### ISSUE THREAD

**Issue by AnonymousWP** [yogeshojha/rengine#1080]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I created a new user to test if I could login with the new user. I tried so by creating the user, logging out and then logging in again with the just-created user's credentials. However, this causes an error 500.

### Expected Behavior

I'd expect it to login properly without returning an error. This is likely caused because the login path differs (see below).

### Steps To Reproduce

1. Go to https://127.0.0.1/default/admin_interface/ to create the user
2. Logout: https://127.0.0.1/logout/
3. You end up at https://127.0.0.1/login/
4. Use the credentials of the user you just created
5. Notice the error 500

I think it has to do with /login, because logging in with the existing user also doesn't work. Usually, the login path is https://127.0.0.1/login/?next=/.

### Environment

```markdown
- reNgine: 2.0.1
- OS: Windows 10
- Python: 3.10
- Docker Engine: v24.0.6
- Docker Compose: v2.23.0
- Browser: Google Chrome 119.0.6045.160
```

### Anything else?

I can only provide logs tomorrow. If needed, I'll provide them.

**Comment by AnonymousWP** [comment]
I obviously updated, but the issue persists, so I think your PR is not directly related to my issue.

**Comment by psyray** [comment]
Here is the full error
![image](https://github.com/yogeshojha/rengine/assets/1230954/5cb7bd64-618e-4387-9747-6e98decffa64)

**Comment by psyray** [comment]
Fixed in #1124
